client/src/watchParty.tsx

```diff
@@ -1,40 +1,41 @@
 import axios from 'axios';
 import { useState, useEffect } from 'react';
 import { Card } from 'react-bootstrap';
-const WatchParty = ({ videos }) => {
-  const [video, setVideo] = useState(() => videos ? videos[0] : {})
-  let config = {
+
+function WatchParty({ videos }) {
+  const [video, setVideo] = useState(() => (videos ? videos[0] : {}));
+  const config = {
     method: 'get',
     url: 'http://localhost:4040/party/',
-    headers: { }
+    headers: {},
   };
```

Comment:
make sure you're using relative endpoints for internal APIs:
`axios.get('/party')`